src/email/serverless.yml

```diff
@@ -27,13 +28,13 @@ custom:
     lambdaPort: 3002
     httpPort: 3000
   isManagedDomain:
-    false: 'true' # To negate the environment variable DISABLE_DOMAIN
-    true: 'false' # if you know of a way to do conditionals please change this
+    false: "true" # To negate the environment variable DISABLE_DOMAIN
+    true: "false" # if you know of a way to do conditionals please change this
   customDomain:
-    domainName: "${env:DOMAIN}" # Change this to your domain.
+    domainName: ${env:DOMAIN, "example.com"} # Change this to your domain.
     basePath: "email" # This will be prefixed to all routes
     stage: ${self:provider.stage}
-    certificateName: "${env:CERTIFICATE_DOMAIN}"
+    certificateName: ${env:CERTIFICATE_DOMAIN, "example.com"}
     createRoute53Record: true
```

Comment:
need `example.com`? or can omit

Reply:
currently its having a default fall back of "example.com" if we dont have that we can either give another string url to fallback on or we can add it in the .env file. But currently the .env file doesn't have so i dont think it matters on local.